pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -27,10 +27,15 @@ const (
 	libvirtDriverCommand     = "crc-driver-libvirt"
 	libvirtDriverDownloadURL = "https://github.com/code-ready/machine-driver-libvirt/releases/download/0.9.1/crc-driver-libvirt"
 	defaultPoolSize          = "20 GB"
+	crcDnsmasqConfigFile     = "crc.conf"
 )
 
 var (
 	libvirtDriverBinaryPath = filepath.Join(driverBinaryDir, libvirtDriverCommand)
+	crcDnsmasqConfigPath    = filepath.Join("/", "etc", "NetworkManager", "dnsmasq.d", crcDnsmasqConfigFile)
```

Comment:
Don't use hard coded separator. `filepath.Join(string(filepath.Separator),"A","B","C")`

- https://golang.org/pkg/path/filepath/#pkg-constants

Reply:
well, this is already Linux-only, but sure.